antarest/study/storage/rawstudy/model/filesystem/config/files.py

```diff
@@ -78,10 +152,10 @@ def build(study_path: Path, study_id: str, output_path: t.Optional[Path] = None)
     Returns:
         An instance of `FileStudyTreeConfig` filled with the study data.
     """
-    is_zip_file = study_path.suffix.lower() == ".zip"
+    is_archive = study_path.suffix.lower() in {".zip", ".7z"}
```

Comment:
We should not hard code in this file the supported archive formats. This will be difficult to maintain.

We should have a constant and a method `is_archive` in a common module `antarest.core.utils.archives`